utils/python/CIME/case_run.py

```diff
@@ -171,7 +171,8 @@ def resubmit_check(case):
     elif dout_s and mach == 'mira':
         caseroot = case.get_value("CASEROOT")
         cimeroot = case.get_value("CIMEROOT")
-        cmd = "ssh cooleylogin1 'cd %s; CIMEROOT=%s ./case.submit %s --job case.st_archive' "%(caseroot, cimeroot, caseroot)
+        cmd = "ssh cooleylogin1 'cd %s; CIMEROOT=%s ./case.submit %s"%(caseroot, cimeroot, caseroot)
+        cmd = cmd + " --job case.st_archive' "
```

Comment:
Are you using code_checker? If pylint is flagging lines like this then you must be running it with more checks than we typically use.